tests/acceptance/core/testcase/upgradecluster.go

```diff
@@ -0,0 +1,97 @@
+package testcase
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/k3s-io/k3s/tests/acceptance/shared/util"
+	"github.com/onsi/ginkgo/v2"
+)
+
+// TestUpgradeClusterManually upgrades the cluster "manually"
+func TestUpgradeClusterManually(g ginkgo.GinkgoTestingT, version string) error {
```

Comment:
Theses upgrade functions feel highly specific and would likely make more sense being defined within the upgrademanual_test.go file itself.